src/installer/installer.go

```diff
@@ -231,6 +241,19 @@ func (i *installer) extractIgnitionToFS(ignitionPath string) (err error) {
 	return err
 }
 
+func (i *installer) generateSshKeyPair() error {
+	i.log.Info("Generating new SSH key pair")
+	if err := i.ops.Mkdir(sshDir); err != nil {
+		i.log.Errorf("Failed to create SSH dir: %s", err)
+		return err
+	}
+	if _, err := i.ops.ExecPrivilegeCommand(utils.NewLogWriter(i.log), "ssh-keygen", "-q", "-f", sshKeyPath, "-N", ""); err != nil {
+		i.log.WithError(err).Error("Failed to generate SSH key pair")
```

Comment:
Be consistent 
o.log.Errorf / i.log.WithError(err).Error

Reply:
I missed that one when I replaced all `log.Errorf` with `log.WithError(err).Error()`, thanks.